org/pr/installable-build.ts

```diff
@@ -123,9 +123,12 @@ async function getDownloadCommentText(status) {
     }
   }
 
-  const apkArtifact = artifacts.find(artifact => artifact.path.endsWith(".apk"))
-  if (apkArtifact) {
-    return `You can test the changes on this Pull Request by downloading the APK [here](${apkArtifact.url}).`
+  const apkArtifacts: Array<any> = artifacts.filter(artifact => artifact.path.endsWith(".apk"))
+  if (apkArtifacts.length == 1) {
+    return `You can test the changes on this Pull Request by downloading the APK [here](${apkArtifacts[0].url}).`
+  } else if (apkArtifacts.length > 1) {
+    const links = apkArtifacts.map(artifact => `- [${artifact.path.split("/").pop()}](${artifact.url})`).join(`\n`)
+    return `You can test the changes on this Pull Request by downloading the APKs:\n${links}.`
```

Comment:
Nit: wdyt about removing the `.` at the end since I kinda feel it will look weird when we have multiple lines and only the last one has the period?

Reply:
Yep, this was an overlook after copy/pasting from the wording of the single-APK case. Will fix, thanks!